closure/repositories.bzl

```diff
@@ -69,6 +69,8 @@ def closure_repositories(
         omit_org_ow2_asm_tree = False,
         omit_org_ow2_asm_util = False,
         omit_phantomjs = False,
+        omit_rules_proto = False,
+        omit_rules_python = False,
```

Comment:
Please also include `rules_{cc,java}`. They are required by the native rules, so they are implicit dependencies, but I'm not sure how long they'll stay that way.

Reply:
I would rather not add them in this PR, but prefer to add the later. In Gerrit we have also consumed Java rules from Starlark, and not fetched the rules in WORKSPACE file.

Reply:
OK, I think they'll be available at least during `1.x`, so that shouldn't be an issue for the next 3+ month.

Reply:
Actually, I think `rules_{cc,java}` is only available in Bazel 0.29+. Please add them in this PR.

With Bazel 0.28.1, I'm seeing the following error:

```
ERROR: error loading package 'closure/templates': Unable to find package for @rules_java//java:defs.bzl: The repository '@rules_java' could not be resolved.
INFO: Elapsed time: 0.211s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (3 packages loaded)
```

Reply:
I've added `rules_{cc,java}`. Done.